gap/PolygonalComplexes/constructors.gd

```diff
@@ -980,6 +980,38 @@ DeclareOperation( "UmbrellaDescriptorOfSurface", [IsSimplicialSurface] );
 #
 #! @BeginGroup
 #! @Description
+#! Let <A>surf</A> be a simplicial surface. This method returns a list, <A>umbtipdesc</A>, of 
+#! cyclic permutations and lists of integers like the umbrella descriptor of a surface. 
+#! For the tip decriptor however the set of vertices (instead of the set of faces) of <A>surf</A> 
```

Comment:
The convention in the GAP help is to describe a function without knowledge of any other functions. Therefore, it might be better if you could document UmbrellaTipDescriptor without assuming that the user is familiar with UmbrellaDescriptor. Please feel free to copy the text from UmbrellaDescriptor and modify it accordingly, if that helps.